src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -403,11 +405,22 @@ export class SimpleSuggestWidget<TModel extends SimpleCompletionModel<TItem>, TI
 		this._persistedSize.reset();
 	}
 
+	showTriggered(auto: boolean, delay: number, cursorPosition: { top: number; left: number; height: number }) {
```

Comment:
Should we stay consistent and track explicitly invoked here too? Inverting to auto is a little ambiguous